nethserver-subscription.spec

```diff
@@ -39,7 +37,7 @@ cp -a lib/nethserver_alerts.py root%{python2_sitelib}
 
 %install
 (cd root; find . -depth -print | cpio -dump %{buildroot})
-%{genfilelist} %{buildroot} > filelist
+%{genfilelist} --ignoredir '/opt' --ignorefile '/etc/cron.daily/nethserver-inventory' %{buildroot} > filelist
```

Comment:
I think we can totally remove the `/etc/facter/facts.d` but it should be tested.

Reply:
Confirmed, we can remove the directory.